core/types/receipt.go

```diff
@@ -142,6 +186,21 @@ type v3StoredReceiptRLP struct {
 	GasUsed         uint64
 }
 
+// LegacyBobaReceiptRLP is the pre bedrock storage encoding of a
+// receipt. It will only exist in the database if it was migrated using the
+// migration tool. Nodes that sync using snap-sync will not have any of these
```

Comment:
It would definitely be cool if we could get these to sync through the Erigon state replication stuff (outside the scope of this work though)